src/main/java/bio/terra/service/dataset/flight/create/DatasetCreateFlight.java

```diff
@@ -57,8 +59,7 @@ public DatasetCreateFlight(FlightMap inputParameters, Object applicationContext)
 
         // Google says that ACL change propagation happens in a few seconds, but can take 5-7 minutes. The max
         // operation timeout is generous.
-        RetryRuleExponentialBackoff pdaoAclRetryRule =
-            new RetryRuleExponentialBackoff(2, 30, 600);
+        RetryRuleExponentialBackoff pdaoAclRetryRule = getDefaultExponentialBackoffRetryRule();
```

Comment:
Is it required to declare this as `RetryRuleExponentialBackoff`? I would expect `RetryRule` here instead, just as we use `List` not `ArrayList` for declarations.

Reply:
Fair point.  I was leaving the current style.  I think we tend to declare to more explicit classes (though I'm happy to change this to a RetryRule)